app/scripts/modules/core/src/instance/loadBalancer/InstanceLoadBalancerHealth.tsx

```diff
@@ -43,17 +47,35 @@ export class InstanceLoadBalancerHealth extends React.Component<IInstanceLoadBal
       </div>
     );
 
+    const healthCheckLinkSpan = (
+      <span className="pad-left small">
+        <a
+          ng-if="targetGroup.healthCheckPath"
+          target="_blank"
+          href={`${healthCheckProtocol}://${ipAddress}${healthCheckPath}`}
+        >
+          Health Check
+        </a>
+      </span>
+    );
+
     // Only wrap with tooltip if we have text for a tooltip
     const tooltipText = healthState !== 'Up' ? description : '';
     if (tooltipText) {
       const tooltip = <Tooltip id={name}>{tooltipText}</Tooltip>;
       return (
         <OverlayTrigger placement="left" overlay={tooltip}>
           {healthDiv}
+          {ipAddress && healthCheckPath && healthCheckLinkSpan}
         </OverlayTrigger>
       );
     }
 
-    return healthDiv;
+    return (
+      <React.Fragment>
```

Comment:
I think we generally only use `<React.Fragment>` if we need to add a `key` prop.  We use the short version most other places: `<>`